src/Compilers/CSharp/Test/Symbol/Symbols/SymbolErrorTests.cs

```diff
@@ -3470,6 +3470,25 @@ public void CS0246ERR_SingleTypeNameNotFound06()
                 );
         }
 
+        [Fact, WorkItem(69700, "https://github.com/dotnet/roslyn/issues/69700")]
+        public void CS0246ERR_SingleTypeNameNotFound07()
+        {
+            var text =
+@"class C
+{
+    [SomeAttribute<int>] // Error should occur here
+    static void Foo()
+    {
+    }
+}
+";
+            CreateCompilation(text).
+                VerifyDiagnostics(
```

Comment:
nit: if you run the test with empty (or incorrect) expected diagnostics, the test failure output will give you a block of text that can be copy&pasted. It'll include a human-readable comment, like in the test above (line 3467). That's slightly nicer, as it shows where the error occurs and what the error message says.